WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -127,6 +128,31 @@ public static void showStockMediaPickerForResult(Activity activity,
         activity.startActivityForResult(intent, requestCode);
     }
 
+    public static void continueJetpackConnect(Context context, Source source, SiteModel site) {
+        switch (source) {
+            case NOTIFICATIONS:
+                continueJetpackConnectForNotifications(context, site);
+                break;
+            case STATS:
+                continueJetpackConnectForStats(context, site);
+                break;
+        }
+    }
+
+    private static void continueJetpackConnectForNotifications(Context context, SiteModel site) {
+        Intent intent = new Intent(context, WPMainActivity.class);
+        intent.putExtra(WordPress.SITE, site);
+        intent.putExtra(WPMainActivity.ARG_CONTINUE_JETPACK_CONNECT, true);
+        context.startActivity(intent);
+    }
+
+    private static void continueJetpackConnectForStats(Context context, SiteModel site) {
+        Intent intent = new Intent(context, StatsConnectJetpackActivity.class);
```

Comment:
Does it make sense to directly start `JetpackConnectionWebViewActivity.startJetpackConnectionFlow` from here? The only thing we do in `StatsConnectJetpackActivity` is that we call this `onCreate`.

Reply:
We could call `JetpackConnectionWebViewActivity.startJetpackConnectionFlow`, but I chose to use `StatsConnectJetpackActivity` to keep the flow consistent with the other Stats Jetpack connect flow.  That way we have one point of reference for Jetpack connect via Stats.  Do you see any downside to using `StatsConnectJetpackActivity` here?

Reply:
Not really, it's just little harder to understand the flow but I'm fine with that.